tests/testsuite/build_script_extra_link_arg.rs

```diff
@@ -303,8 +314,16 @@ fn link_arg_with_doctest() {
 
     p.cargo("test --doc -v")
         .without_status()
-        .with_stderr_contains(
-            "[RUNNING] `rustdoc [..]--crate-name foo [..]-C link-arg=--this-is-a-bogus-flag[..]",
+        .with_stderr_data(
+            "\
+[COMPILING] foo v0.0.1 ([ROOT]/foo)
+[RUNNING] `rustc --crate-name build_script_build --edition=2015 build.rs [..]
+[RUNNING] `[ROOT]/foo/target/debug/build/foo-[HASH]/build-script-build`
+[RUNNING] `rustc --crate-name foo --edition=2015 src/lib.rs [..]
+[FINISHED] `test` profile [unoptimized + debuginfo] target(s) in [ELAPSED]s
+[DOCTEST] foo
```

Comment:
Like line 24 in this file, maybe this could be simplified like:

```
...
[RUNNING] `rustdoc [..]--crate-name foo [..]-C link-arg=--this-is-a-bogus-flag[..]
...
```